apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,31 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs)
+  : m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs)
+{
+  m_servoPwmOut.period_ms(20);
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees)
+{
+  if (degrees <= m_servoRangeInDegrees && degrees >= 0 && m_minPulsewidthInMs + degrees/m_servoRangeInDegrees <= m_maxPulsewidthInMs)
+  {
+    m_servoPwmOut.pulsewidth_ms(m_minPulsewidthInMs + degrees/m_servoRangeInDegrees);
```

Comment:
What if m_maxPulsewidthInMs is not 2ms? You'll want to map degrees to a pulsewidth value between m_minPulsewidthInMs and m_maxPulsewidthInMs